### PR TITLE
Attempt at a more natural Placeholder API

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

Attempt at a more natural Placeholder API. Use Chip with empty strings instead of PlaceholderChip. 

#### WHY

Use the PlaceholderState API as John intended.

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt [hunk 1]
```diff
@@ -360,6 +380,34 @@ public fun Chip(
     }
 }
 
+@Composable
+fun Modifier.placeholderIf(
```
Comment (garanj) [inline compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt:384]: Cool!

**Review by garanj — APPROVED** [review]
Looks good!

**Comment by yschimke** [comment]
@garanj I'll add it to all the material Composables, and then circulate the screenshots.

File: compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_CompactChipTest_mirrored.png
Comment (yschimke) [inline compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_CompactChipTest_mirrored.png:1]: Fix before landing

**Pull request closed without merging**